openpype/hosts/resolve/plugins/publish/precollect_workfile.py

```diff
@@ -15,6 +15,8 @@ class PrecollectWorkfile(pyblish.api.ContextPlugin):
     def process(self, context):
 
         asset = get_current_asset_name()
+        # AYON compatibility split name and use last piece
```

Comment:
We are not checking if `AYON_SERVER_ENABLED`  ?

Reply:
Well, it does not hurt in OpenPype ("sh0010" will stay "sh0010"), but I will add it.